scripts/release-fdroid.sh

```diff
@@ -0,0 +1,78 @@
+version_name=$(cat VERSION)
+working_dir="/tmp/fdroid-release"
+clone_dir="/tmp/fdroid-release/$version_name"
+username=$USERNAME
+metadata_file="$clone_dir/metadata/im.status.ethereum.yml"
+release_link=$RELEASE_LINK
+apk_file="$working_dir/$version_name.apk"
+commit_sha=$(git rev-parse --verify HEAD)
+
+echo "recreating working directory..."
+
+rm -rf $working_dir
+mkdir -p $working_dir
+
+echo "fetching release..."
+
+wget $release_link -O $apk_file
+
+echo "cloning branch, this might take a while..."
+
+git clone git@gitlab.com:$username/fdroiddata.git $clone_dir
+
+echo "updating branch with upstream, this also might take a while..."
+
+cd $clone_dir \
+  && git remote add upstream https://gitlab.com/fdroid/fdroiddata.git \
+  && git fetch upstream && git checkout master \
+  && git reset --hard upstream/master
+
+echo "creating release branch..."
+
+branch_name="release/$version_name"
+git checkout -b $branch_name
+
+echo "extracting release code..."
+
+version_code=$(apkanalyzer manifest version-code $apk_file)
+
+# grep last versionName line and extract line number
+start_line_raw=$(grep -n "versionName:" $metadata_file | tail -n1 | grep -o -P '([0-9]+):')
+start_line=${start_line_raw::-1}
+
+# grep last build line and extract line number
+end_line_raw=$(grep -n "build:" $metadata_file | tail -n1 | grep -o -P '([0-9]+):')
+end_line=${end_line_raw::-1}
```

Comment:
This stuff would be so much nicer if we just used Python. We do have it in the default shell:
https://github.com/status-im/status-react/blob/710e55002b29bb3a9144d322f195c0d3e9fc0dd1/nix/shell.nix#L20-L21